packages/babel-preset-env/src/normalize-options.js

```diff
@@ -48,11 +46,7 @@ const getValidIncludesAndExcludes = (
   new Set([
     ...allPluginsList,
     ...(type === "exclude" ? modulePlugins : []),
-    ...(corejs
-      ? corejs == 2
-        ? [...Object.keys(corejs2Polyfills), ...defaultWebIncludes]
```

Comment:
I like the idea of _not_ applying web polyfills for node-only targets. However these `defaultWebIncludes` are applied _unconditionally_. That is also why the core-js 3 support does not have `defaultWebIncludes`. I think we should bring back and properly filter out web-only polyfills for node-only targets. Surely it falls into a new PR.